src/filters/tags.js

```diff
@@ -43,9 +42,9 @@ params:
 */
 const displayName = (tag) => {
   const capitalize = ([first, ...rest]) =>
-    first ? first.toUpperCase() + rest.join('') : '';
+    first && first.toUpperCase() + rest.join('');
```

Comment:
Won't this change mean we now get `false` instead of `''` if tag is empty?

Reply:
Yup you are right.  Maybe I'm misunderstanding how this is expected to called; can't seem to get the coverage for returning an empty string :woman_shrugging: